using-timescaledb/continuous-aggregates.md

```diff
@@ -0,0 +1,249 @@
+# Continuous Aggregates 
+TimescaleDB continuous aggregates allow users to speed aggregate queries which
+span large amounts of time by continuously calculating them in the background
+and materializing the aggregates in a separate view. Conntinuous aggregates use
+the background worker framework to run aggregates on a schedule as new data is
+inserted into a hypertable or when old data is modified. There's a bit more
+complexity to how they're implemented which you can read about in our [blog post
+announcing the feature][continuous-aggs-blog], but for now, suffice it to say
+that they can really help if you've been having trouble with long-running
+aggregate queries. 
+
+
+### Creating a Continuous Aggregate View [](create)
+Continuous aggregates are exposed to the user by extending the PostgreSQL
+[`CREATE VIEW`][postgres-createview] command using a `WITH` option, must contain
+a [`time_bucket`][time-bucket] function call in the group by clause and support
+all parallel-safe aggregates. 
+
+Let's suppose we have a hypertable `device_readings` created as so:
+```sql
+CREATE TABLE device_readings (
+      observation_time  TIMESTAMPTZ       NOT NULL,
+      device_id         TEXT              NOT NULL,
+      metric            DOUBLE PRECISION  NOT NULL,
+      PRIMARY KEY(observation_time, device_id)
+);
+SELECT create_hypertable('device_readings', 'observation_time');
+```
+
+And we want to query per-device readings aggregated on an hourly basis, we might
+create a continuous aggregate as so:
+
+```sql
+CREATE VIEW device_summary
+WITH (timescaledb.continuous) --This flag is what makes the view continuous
```

Comment:
the sql comments should be deleted and should be in text below

Reply:
I think they're useful and far easier to parse this way.